README.md

```diff
@@ -3,6 +3,7 @@
 The **[Amazon DynamoDB][ddb] Client-side Encryption in Java** supports encryption and signing of your data when stored in Amazon DynamoDB.
 
 A typical use of this library is when you are using [DynamoDBMapper][ddbmapper], where transparent protection of all objects serialized through the mapper can be enabled via configuring an [AttributeEncryptor][attrencryptor].
+**Please note that it is critically important that you use `SaveBehavior.CLOBBER` when using AttributeEncryptor.**
```

Comment:
Can we compact the wording of the warning for grokkability, and add an expanded explanation?

Perhaps:

> **Important: use `SaveBehavior.CLOBBER` with `AttributeEncryptor`.** This is because ...

Should https://github.com/aws/aws-dynamodb-encryption-java/blame/ded8364a9baee731f6f83ddddca8cec5bc614f9c/README.md#L77 be `DynamoDBMapperConfig.CLOBBER` as well? It looks to my quick read like it is affected, since it's using the `AttributeEncryptor`.